lib/index.ts

```diff
@@ -10,12 +10,15 @@ const hostIPList = getLocalhostAddress();
 logger.info(`🎉 RSSHub is running on port ${port}! Cheers!`);
 logger.info('💖 Can you help keep this open source project alive? Please sponsor 👉 https://docs.rsshub.app/sponsor');
 logger.info(`🔗 Local: 👉 http://localhost:${port}`);
-for (const ip of hostIPList) {
-    logger.info(`🔗 Network: 👉 http://${ip}:${port}`);
+if (config.listenInaddrAny) {
+    for (const ip of hostIPList) {
+        logger.info(`🔗 Network: 👉 http://${ip}:${port}`);
+    }
 }
 
 const server = serve({
     fetch: app.fetch,
+    hostname: config.listenInaddrAny ? null : '127.0.0.1',
```

Comment:
```suggestion
    hostname: config.listenInaddrAny ? undefined : '127.0.0.1',
```

Reply:
Thanks, looks better and works - updated.